packages/sw-runtime-caching/src/lib/network-first.js

```diff
@@ -13,13 +13,19 @@
  limitations under the License.
 */
 
+import {CacheableResponse} from '../../../sw-cacheable-response/src/index';
 import Handler from './handler';
 import assert from '../../../../lib/assert';
 
 /**
  * An implementation of a [network first](https://developers.google.com/web/fundamentals/instant-and-offline/offline-cookbook/#network-falling-back-to-cache)
  * request strategy.
  *
+ * By default, [opaque responses](http://stackoverflow.com/questions/39109789/what-limitations-apply-to-opaque-responses)
```

Comment:
I wonder if there's value using the shorter http://stackoverflow.com/q/39109789 form for links that are going to appear in source but will defer to your judgement call.